tests/test_inmemorychannel.py

```diff
@@ -62,7 +62,6 @@ async def test_multi_send_receive(channel_layer):
     """
     Tests overlapping sends and receives, and ordering.
     """
```

Comment:
Are you confident that the test suite covers all these changes? Are there any missing test-cases that we should add?

My particular areas of concern: the `del` -> `.pop` functionality, and `ChannelFull` handling, changes in queue capacity

Reply:
I agree, more tests would be better. I just invested no more time after getting no feedback long time

Reply:
Sorry about that, we're trying to do a better job of responding to issues/PRs. If you find time, please consider other test cases

Reply:
- I added a test for the race.
- changes in queue capacity are not happening (they are compiled statically in constructor) and if wanted we need to rewrite with a lock.
- ChannelFull is tested already